templates/careers/_search.html

```diff
@@ -7,56 +7,69 @@
       showAutoComplete(search_term);
     });   
     showAutoComplete("");
-    
     const urlParams = new URLSearchParams(window.location.search);
     const querySearchText = urlParams.get("search");
-    
     ShowJobList(querySearchText)
   });
   
   const ShowJobList = (search_term) => {
-    const ul = document.querySelector(".js-job-list");
+    const containerDiv = document.querySelector(".js-job-list");
     let searchResult = search_term ? fuse.search(search_term).map( a => a.item ) : vacancies
     let jobs = vacancies.filter( a => searchResult.includes(a) )
-    ul.innerHTML = ''
+    containerDiv.innerHTML = ''
     jobs.forEach( job => { 
-      const li= document.createElement('li')
-      li.setAttribute('class', 'p-list__item');
-      li.dataset.office = job.office
-      li.dataset.employment =  job.employment
-      li.dataset.location = job.location
-      li.dataset.management = job.management
-      li.dataset.sector = job.departments
+      const stripDiv = document.createElement('div')
+      stripDiv.setAttribute('class', 'p-strip is-shallow is-bordered u-no-padding--top');
+      stripDiv.dataset.office = job.office
+      stripDiv.dataset.employment =  job.employment
+      stripDiv.dataset.location = job.location
+      stripDiv.dataset.management = job.management
+      stripDiv.dataset.sector = job.departments
+      
+      const parentDiv = document.createElement('div')
+      parentDiv.setAttribute('class', 'row ');
+
+      const leftCol = document.createElement('div')
+      leftCol.setAttribute('class', 'col-4');
+      
+      const rightCol = document.createElement('div')
+      rightCol.setAttribute('class', 'col-5');
 
       const location = document.createElement('p')
-      location.setAttribute('class', 'u-no-margin--bottom u-sv1');
-      const small= document.createElement('small')
-      small.textContent = job.location
-
-      const title = document.createElement('h2')
-      title.setAttribute('class', 'p-heading--3 u-no-padding--top u-no-margin--bottom');
-      const anchor = document.createElement('a')
-      anchor.setAttribute('class', 'p-link--soft');
-      anchor.setAttribute('href', `/careers/${job.id}`)
-      anchor.innerHTML= `${job.title}&nbsp;&rsaquo;`
+      location.setAttribute('class', 'u-text--muted');
+      location.textContent = job.location
+
+      const title = document.createElement('p')
```

Comment:
I think this should be a `h3` with `p-heading--4` class (rather than giving that class to the anchor tag on line 44) - I would expect to see the job titles listed in the headings heirarchy.